src/rsz/src/RepairDesign.cc

```diff
@@ -851,7 +851,8 @@ void RepairDesign::repairNetWire(
                  level,
                  units_->distanceUnit()->asString(dbuToMeters(wire_length), 1),
                  units_->distanceUnit()->asString(dbuToMeters(max_length_), 1));
-      split_length = min(split_length, max(max_length_ - wire_length_ref, 0));
+      split_length = min(max_length_, length / 2);
```

Comment:
This reintroduces what was fixed in 3b97c566

Reply:
@arthurjolo please comment

Reply:
I believe this was overwritten during a merge. But I think it easy to fix: `split_length = min(length / 2, max(max_length_ - wire_length_ref, 0));`
This should account for both changes, @povik do you agree?

Reply:
Yes, that looks good to me! (I was lazy to type that out myself...)

Reply:
@arthurjolo please make a PR for your fix.